admin/tabs/cluster/instances/websocket/WebSocketColumns.ts

```diff
@@ -73,11 +73,33 @@ export const lastReceivedTime: ColumnSpec = {
     width: 140
 };
 
-export const clientAppVersion: ColumnSpec = {
+export const appVersion: ColumnSpec = {
     field: {
-        name: 'clientAppVersion',
+        name: 'appVersion',
```

Comment:
As the list of data we sent grows, I wanted a cleaner way to add to the list of props. It seems easiest to just keep the param names matching the props on XH, so we can:
https://github.com/xh/hoist-react/pull/3966/files#diff-ef983727de9882e21a65a8e308ab6d76be6ca55458bbabc2b4a0008e633125b9R308